frontend/src/store/choices.ts

```diff
@@ -0,0 +1,30 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export interface Choice {
+  id: number;
+  name: string;
+  lesson_id: number;
+  is_correct: boolean;
+}
+
+interface ChoiceState {
+  choices: Choice[],
+}
+
+const initialState: ChoiceState = {
+  choices: [],
+};
+
+export const choiceSlice = createSlice({
+  name: 'choice',
+  initialState: initialState,
+  reducers: {
+    getChoices: (state, action) => {
+      // state.choices = [...state.choices, ...action.payload];
```

Comment:
Remove

Reply:
will delete this file, miss since there's no need for a choice reducer anymore